.gitignore

```diff
@@ -2,5 +2,6 @@
 .vscode/
 Cargo.lock
 gitoid.h
-c-test
+gitoid/test/c_test
```

Comment:
not directly related to the rename of gitbom to omnibor, but I noticed this was missing when running tests

Reply:
Oh great! I'd just noticed the same thing this morning.